ui/src/map/layers/utils/alignment-layer-utils.ts

```diff
@@ -31,6 +28,33 @@ export const NORMAL_ALIGNMENT_OPACITY = 1;
 export const REFERENCE_LINE_ALIGNMENT_WIDTH = 3;
 export const LOCATION_TRACK_ALIGNMENT_WIDTH = 1;
 
+type AlignmentStyleType = 'IN_USE' | 'NOT_IN_USE' | 'BUILT_BACKGROUND' | 'BUILT';
```

Comment:
Tässä tehty uusi tyyppi, sillä näitä käytetään käytännössä pelkästään Z-indeksilaskentaan ja Rakennettu-tilaisen valkoinen tausta tarvitsee oman z-indeksin